pkgs/applications/terminal-emulators/blackbox-terminal/default.nix

```diff
@@ -62,7 +64,17 @@ stdenv.mkDerivation rec {
   ];
   buildInputs = [
     gtk4
-    vte-gtk4
+    (vte-gtk4.overrideAttrs (old: lib.optionalAttrs sixelSupport {
```

Comment:
option should not affect src, but only sixel support

Reply:
You mean we should always use the same src?

Reply:
> 
>     make blackbox-terminal always use version of vte that's listed in Flatpack
>     make sixel support optional flag, that's disabled by default
>     need approval from blackbox-terminal maintainer
>